rxnorm-integration/src/test/java/dev/ikm/tinkar/rxnorm/integration/AbstractIntegrationTest.java

```diff
@@ -76,26 +84,50 @@ protected static String findFilePath(String baseDir, String fileKeyword) throws
      * @return File status, either Found/NotFound
      * @throws IOException
      */
-    protected int processFile(String sourceFilePath, String errorFile) throws IOException {
+    protected int processOwlFile(String sourceFilePath, String errorFile) throws IOException {
         int notFound = 0;
-        try (BufferedReader br = new BufferedReader(new FileReader(sourceFilePath));
-             BufferedWriter bw = new BufferedWriter(new FileWriter(errorFile))) {
-            String line;
-            while ((line = br.readLine()) != null) {
-                if (line.startsWith("id") || line.startsWith("alternateIdentifier")) continue;
-                if (!assertLine(line.split("\\t"))) {
+        String owlContent = readFile(sourceFilePath);
+        try (BufferedWriter bw = new BufferedWriter(new FileWriter(errorFile))) {
+            // Process the OWL content to extract class declarations and annotations
+            List<RxnormData> rxnormConcepts = RxnormUtility.extractRxnormData(owlContent);
+            LOG.info("Found " + rxnormConcepts.size() + " class declarations in the OWL file");
+
+            // Compare concepts for each class
+            for (RxnormData rxnormConcept : rxnormConcepts) {
+                if (!assertOwlElement(rxnormConcept)) {
                     notFound++;
-                    bw.write(line);
+                    System.out.print("rxnormConcept.toString(): " + rxnormConcept.toString());
```

Comment:
This is fine for now, but this system out should be removed in the follow on story to fix these null id concepts